frontend/src/components/pages/profile/EditUser.tsx

```diff
@@ -77,14 +76,7 @@ export const EditUser: React.FC<EditUserProps> = ({ open, user, onSubmit, onClos
   const handleSubmit = () => {
     updateUser({
       variables: { userData: updateUserInput },
-      update: (cache, { data }) => {
-        if (!data || !data.updateUser || !data.updateUser.user) {
-          return;
-        }
-        cache.writeQuery<User>({ query: GET_USER, data: data.updateUser.user });
-        onSubmit();
-      },
-    });
```

Comment:
These consistently raised errors due to missing fields. Apollo resolves this automagically with [Store normalized objects](https://www.apollographql.com/docs/react/caching/overview/#4-store-normalized-objects) 🧙‍♀️